### ISSUE TITLE
Omit undefined values from query

### ISSUE THREAD

**Issue by minht11** [43081j/picoquery#46]
I am using `qs` library and it seems to omit query entirely when its' value is undefined.

```ts
import * as qs from 'qs';
import * as pico from 'picoquery';

const data = {
  publicationState: undefined,
};

const query = qs.stringify(data, {
  encodeValuesOnly: true,
});

const newQuery = pico.stringify(data, {
  nestingSyntax: 'index',
});

console.log(query);
console.log(newQuery);
```
Result:
QS: ``
Picoquery: `publicationState=`

[Reproduction](https://stackblitz.com/edit/vitejs-vite-zrdrss?file=src%2Fmain.ts)

**Comment by 43081j** [comment]
Makes sense to me. Should be an easy fix if you want to contribute

**Comment by 43081j** [comment]
Fixed in #51
